Kernel/Process.cpp

```diff
@@ -2726,6 +2727,28 @@ int Process::sys$reboot()
     return ESUCCESS;
 }
 
+int Process::sys$mount(int devmajor, int devminor, const char* mountpoint)
+{
+    auto* dev = VFS::the().get_device(devmajor, devminor);
+    auto* disk_device = static_cast<DiskDevice*>(dev);
+
+    dbgprintf("mount: attempting to mount %d,%d to %s...\n", devmajor, devminor, mountpoint);
+
+    if(dev == nullptr)
```

Comment:
Missing space after `if` here.
Also, we should null-check this pointer as soon as possible, so right after we create it above.
Failing early prevents us from mistakenly using values before they've been validated :)